src/chat/Chat.tsx

```diff
@@ -0,0 +1,186 @@
+import React, {useEffect, useState} from 'react';
+import 'react-chat-elements/dist/main.css';
+import './Chat.css';
+// @ts-ignore
+import { MessageList, Input, Button, ChatList } from 'react-chat-elements';
+import config from '../config.json';
+import {withRouter} from "react-router";
+import {connect} from "react-redux";
+import ChatModel from "./types/Chat";
+import Message from "./types/Message";
+
+const Chat = (props : any) => {
+    const [message, setMessage] = useState<string>('');
+    const [messageList, setMessageList] = useState();
+    const [chatList, setChatList] = useState()
+    const [chatEnabled, setChatEnabed] = useState<boolean>(false)
+
+    const loadChats = async () => {
```

Comment:
Error melding, zet er een try catch omheen geef de error weer voor de gebruiker.
Error message methode is sws handige om te hebben voor elke pagina.